pwchem/objects/base.py

```diff
@@ -345,6 +345,7 @@ def __init__(self, **kwargs):
     self._type = pwobj.String(kwargs.get('type', 'Standard'))
 
     self.proteinFile = pwobj.String(kwargs.get('proteinFile', None))  # to be used when each mol has diff receptor
+    self._scorchScore = Float(kwargs.get('scorch2Score', None))
```

Comment:
I'd rather just create the attribute when the protocol is run, otherwise we would keep having more and more columns that are almost always None